bridge/bridgeapp/views.py

```diff
@@ -73,42 +52,34 @@ def post(self, request, category_id, category_slug):
                 for cat_id in cat_ids:
                     category = Category.objects.get(id=cat_id)
                     thread.categories.add(category)
-                id, slug = cat_ids[0], slugify(Category.objects.get(id=cat_ids[0]).type)
+        delete_commented_out_code
+                id, slug = cat_ids[0], slugify(
+                    Category.objects.get(id=cat_ids[0]).type)
+
         return redirect('category', category_id=id, category_slug=slug)
 
-# class BridgeResponseView(View):
+
 class BridgeThreadView(View):
     def get(self, request, thread_id, resp_id):
-        # categories = Category.objects.get(type=category_id)
-        # thread_object = Thread.objects.get(id=thread_id)
-        # print("Here in BridgeResponseView!")
-        # print(thread_object)
-        # response_form=ResponseForm()
         thread = Thread.objects.get(id=thread_id)
         responses = Response.objects.filter(thread=thread)
-        form = InputForm(initial={'body': 'Write a response to the question.' if not resp_id else Response.objects.get(id=resp_id).body})
+        form = InputForm(initial={
```

Comment:
Would it work if you fold it further like so?
```
form = InputForm(initial={
    'body': 'Write a response to the question.' if not resp_id
        else Response.objects.get(id=resp_id).body}
```

Reply:
I'm not sure, this was also an automatic stylistic change